Sources/MongoSwift/ReadConcern.swift

```diff
@@ -2,53 +2,55 @@ import CLibMongoC
 
 /// A struct to represent a MongoDB read concern.
 public struct ReadConcern: Codable {
+    /// Local ReadConcern, see https://docs.mongodb.com/manual/reference/read-concern-local/.
+    public static let local = ReadConcern(.local)
+    /// Available ReadConcern, see https://docs.mongodb.com/manual/reference/read-concern-available/.
+    public static let available = ReadConcern(.available)
+    /// Linearizable ReadConcern, see https://docs.mongodb.com/manual/reference/read-concern-majority/.
+    public static let linearizable = ReadConcern(.linearizable)
+    /// Majority ReadConcern, see https://docs.mongodb.com/manual/reference/read-concern-linearizable/.
+    public static let majority = ReadConcern(.majority)
+    /// Snapshot ReadConcern, see https://docs.mongodb.com/master/reference/read-concern-snapshot/.
+    public static let snapshot = ReadConcern(.snapshot)
+    /// Empty ReadConcern.
+    public static let empty = ReadConcern(nil)
```

Comment:
rather than calling this `empty`, I think `default` or `serverDefault` would be more accurate. Also, if we're going to do this with `ReadConcern`, we should do the same with `WriteConcern`.